packages/client/src/components/Questions.tsx

```diff
@@ -212,25 +213,34 @@ const Questions: React.FC<
   let disableFutureQuestions = false;
 
   // Check which questions are causing the need for a permit
-  // @TODO: We can refactor this and move to checker.js
-  const permitsPerQuestion = [] as boolean[];
-  checker.isConclusive() &&
-    checker.permits.forEach((permit: Permit) => {
-      const conclusionDecision = permit.getDecisionById("dummy");
-      if (
-        conclusionDecision?.getOutput() === imtrOutcomes.NEED_PERMIT ||
-        conclusionDecision?.getOutput() === imtrOutcomes.NEED_CONTACT
-      ) {
+  // @TODO: Move this to `imtr-client`
+  const permitsPerQuestion = [] as string[];
+  checker.permits.forEach((permit: Permit) => {
+    const conclusionDecision = permit.getDecisionById("dummy");
+
+    if (conclusionDecision) {
+      let outcomeType = "";
+      if (conclusionDecision.getOutput() === imtrOutcomes.NEED_CONTACT) {
+        outcomeType = clientOutcomes.NEED_CONTACT;
+      } else if (conclusionDecision?.getOutput() === imtrOutcomes.NEED_PERMIT) {
+        outcomeType = clientOutcomes.NEED_PERMIT;
+      } else if (conclusionDecision?.getOutput() === imtrOutcomes.NEED_REPORT) {
+        outcomeType = clientOutcomes.NEED_REPORT;
+      }
+
+      if (outcomeType) {
         const decisiveDecisions = conclusionDecision.getDecisiveInputs() as Decision[];
```

Comment:
Can you write this like `const decisiveDecisions: Decision[] = conclusionDecision.getDecisiveInputs()` ?

Reply:
Fails:
```
Type 'Input[]' is not assignable to type 'Decision[]'.
  Type 'Input' is not assignable to type 'Decision'.
    Type 'Question' is missing the following properties from type 'Decision': inputs, rules, getMatchingRules, getDecisiveInputs, and 2 more
```

I tried `const decisiveDecisions: Input[] = conclusionDecision.getDecisiveInputs();` but then i'm getting another fail:
```
Property 'getDecisiveInputs' does not exist on type 'Input'.
  Property 'getDecisiveInputs' does not exist on type 'Question'.
```